### PR TITLE
eth2util: add holesky

### PR DESCRIPTION
**Pull request by xenowits**
Add `holesky` to the list of supported networks.

category: feature 
ticket: #2399

### CODE REVIEW

File: eth2util/network.go [hunk 1]
```diff
@@ -50,10 +50,16 @@ var (
 		GenesisForkVersionHex: "0x90000069",
 		GenesisTimestamp:      1655733600,
 	}
+	Holesky = Network{
+		ChainID:               17000,
+		Name:                  "holesky",
+		GenesisForkVersionHex: "0x00017000",
+		GenesisTimestamp:      1694786400,
```
Comment (LukeHackett12) [inline eth2util/network.go:57]: Is this time changing since launch was delayed?

Comment (xenowits) [reply eth2util/network.go:57]: I took these data from the ticket itself. 

Didn't verify it myself

Comment (xenowits) [reply eth2util/network.go:57]: And yeah, i think the timestamp would change to reflect next week's

Comment (xenowits) [reply eth2util/network.go:57]: They haven't updated the README yet - https://github.com/eth-clients/holesky#metadata

Comment (LukeHackett12) [reply eth2util/network.go:57]: According to [this pr](https://github.com/eth-clients/holesky/pull/73/files) it is `1696000704`

Comment (LukeHackett12) [inline eth2util/network.go:57]:
```suggestion
		GenesisTimestamp:      1696000704,
```

Comment (xenowits) [reply eth2util/network.go:57]: yeah cool, let's merge this PR only when the [linked PR](https://github.com/eth-clients/holesky/pull/73/files) is merged

**Review by LukeHackett12 — APPROVED** [review]

File: eth2util/network.go [hunk 2]
```diff
@@ -50,10 +50,16 @@ var (
 		GenesisForkVersionHex: "0x90000069",
 		GenesisTimestamp:      1655733600,
 	}
+	Holesky = Network{
+		ChainID:               17000,
+		Name:                  "holesky",
+		GenesisForkVersionHex: "0x00017000",
+		GenesisTimestamp:      1696000704,
```
Comment (LukeHackett12) [inline eth2util/network.go:57]: They changed it again: 
```suggestion
		GenesisTimestamp:      1695902400,
```

**Pull request merged**